readme.md

```diff
@@ -47,6 +47,7 @@ Please raise a [Pull Request](https://github.com/wizztjh/awesome-webvr/pulls) to
 
 ### Demos
 
+- [ArtSocks](https://art-socks.herokuapp.com/demo.html) - An experiment with VR and web sockets by [dziemid](https://twitter.com/dziemid) (Google Cardboard)
```

Comment:
:+1: Just tested it out with my oculus rift. Very interesting. You are using webvr boilerplate?